meson.build

```diff
@@ -252,10 +252,13 @@ if get_option('coverage') == true
     ]
 endif
 
+hwloc_dep = dependency('hwloc', static : true)
```

Comment:
tests/meson.build has:
```
dep_static = get_option('dependency_link') == 'static'
...
eigen3_dep = dependency('eigen3', static : dep_static)
```

Please move the `dep_static` variable to the top-level meson and then use it here.

Reply:
Or move the variable assignment here.  I think that works too.